internal/backend/user.go

```diff
@@ -101,7 +101,11 @@ func newUser(
 	}
 
 	if err := user.deleteAllMessagesMarkedDeleted(ctx); err != nil {
-		return nil, err
+		logrus.WithError(err).Error("Failed to remove deleted messages")
```

Comment:
I think it might be better to make sure the database is emptied of the messages even if the store deletion fails.

Right now if the store deletion fails, the deleted message wills till remain in the database.

Reply:
I don't think that's the case -- we do the DB delete first inside deleteAllMessagesMarkedDeleted